config-secret/file-system/src/main/resources/application.properties

```diff
@@ -3,7 +3,4 @@ quarkus.openshift.app-secret=app-config
 
 quarkus.s2i.base-jvm-image=registry.access.redhat.com/ubi8/openjdk-11:latest
 
-%test.hello.message=Hello, %s!
-
-# the application binary resides in /home/quarkus with quarkus/ubi-quarkus-native-binary-s2i
-%native.quarkus.openshift.mounts.app-config.path=/home/quarkus/config
```

Comment:
We should probably track these details somewhere upstream or at least document how these paths can be figured out

Reply:
I'm not sure what you mean. I checked the git history and this was added as a workaround for an upstream issue that is now fixed. That's why I removed it.